README.md

```diff
@@ -42,7 +42,17 @@ INFO    -  Cleaning site directory
 [I 160402 15:50:43 handlers:58] Start watching changes
 [I 160402 15:50:43 handlers:60] Start detecting changes
 ```
+If you are using Windows, some of the above commands may not work out-of-the-box.
+A quick solution may be to preface every Python command with `python -m` like this:
```

Comment:
`You may need to inform python to search sys.path for the mkdocs module:`

Reply:
do you want to put this in the readme?

Reply:
yes, the line was a replacement for the "A quick solution" line.